redisson-tomcat/redisson-tomcat-10/src/main/java/org/redisson/tomcat/RedissonSession.java

```diff
@@ -263,7 +263,7 @@ private void fastPut(String name, Object value) {
             return;
         }
         m.fastPut(name, value);
-        if (readMode == ReadMode.MEMORY && this.broadcastSessionUpdates) {
+        if (readMode == ReadMode.MEMORY && this.broadcastSessionUpdates || this.broadcastSessionEvents) {
```

Comment:
broadcastSessionEvents setting is only for `sessionCreated` and `sessionDestroyed` events